REQUIRE

```diff
@@ -1,2 +1 @@
-julia 0.6
-Compat 0.55.0
+julia 0.7-
```

Comment:
Not that it matters much, but should we rather set this to `0.7.0-alpha`?

Reply:
Hmm, now that I think about it I guess this allows early dev versions too. Well, I don't really think anyone would be using anything older than the alpha anyway...